src/languageservice/parser/isKubernetes.ts

```diff
@@ -0,0 +1,7 @@
+import * as Parser from './jsonParser07';
+
+export function setKubernetesParserOption(jsonDocuments: Parser.JSONDocument[], option: boolean): void {
+  for (const jsonDoc in jsonDocuments) {
```

Comment:
`for...in` is designed for `Objects`. While it does work with `Arrays` it is usually better to use `Array.prototype.forEach()` or `for...of`.